### ISSUE TITLE
New Mugs

### ISSUE THREAD

**Issue by shawnroos** [we-are-next/office#11]
Hey @ulandamackay can we get the specs for:

- [ ] The Actual Mug 
- [ ] The Printable Area.

:+1:

**Comment by shawnroos** [comment]
Hey @ulandamackay can we get the specs for:

- [ ] The Actual Mug 
- [ ] The Printable Area.

**Comment by ulandamackay** [comment]
@shawnroos @FrancoRaffa Got a quote for R50 per mug (+ once-off set-up fee of R250) from www.digitalexpress.co.za. That cool?

@shaungards Specs are 200mm by 95mm. They weren't specific about file type.

**Comment by FrancoRaffa** [comment]
Cost is fine.

**Comment by shaungards** [comment]
@FrancoRaffa @ulandamackay are you guys still keen on doing this idea ? 

Heres google drive link 
https://drive.google.com/drive/folders/0Bw86V8wNJWK9V04yZGlaUXlDYjA?usp=sharing

**Comment by shaungards** [comment]
![coffe mug - all togeather](https://cloud.githubusercontent.com/assets/13214973/18553951/db4a27a6-7b62-11e6-9c76-b8961b4eabca.jpg)
 what they will look like when the mugs are lined up

**Comment by FrancoRaffa** [comment]
who did these designs?

**Comment by shaungards** [comment]
@FrancoRaffa I did them, but ages ago

**Comment by FrancoRaffa** [comment]
Then yes. let's do it!

**Comment by shaungards** [comment]
Awesome :) @ulandamackay you think we could still print them at www.digitalexpress.co.za.

**Comment by ulandamackay** [comment]
Yup, definitely can! Will get back in touch with them and check if those files work.

**Comment by ulandamackay** [comment]
@shaungards @FrancoRaffa Shared the files with Digital Express and made an order 👍

**Comment by shaungards** [comment]
@ulandamackay Awesome :)
